releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -2080,13 +2160,12 @@
     "sigs": [
       "cli"
     ],
-    "feature": true,
-    "is_mapped": true
+    "feature": true
   },
   "124598": {
     "commit": "44bd04c0cbddde69aaeb7a90d3bd3de4e417f27f",
     "text": "kubectl describe service and ingress will now use endpointslices instead of endpoints",
-    "markdown": "Kubectl describe service and ingress will now use endpointslices instead of endpoints ([#124598](https://github.com/kubernetes/kubernetes/pull/124598), [@aroradaman](https://github.com/aroradaman)) [SIG CLI and Network]",
+    "markdown": "Kubectl describe service and ingress will now use endpointslices instead of endpoints ([#124598](https://github.com/kubernetes/kubernetes/pull/124598), [@aroradaman](https://github.com/aroradaman))",
```

Comment:
```suggestion
    "markdown": "Kubectl describe service and ingress will now use endpointslices instead of endpoints ([#124598](https://github.com/kubernetes/kubernetes/pull/124598), [@aroradaman](https://github.com/aroradaman))",
```
```suggestion
    "markdown": "Kubectl describe service and ingress will now use endpointslices instead of endpoints ([#124598](https://github.com/kubernetes/kubernetes/pull/124598), [@aroradaman](https://github.com/aroradaman))",
```
```suggestion
    "markdown": "Kubectl describe service and ingress will now use endpointslices instead of endpoints ([#124598](https://github.com/kubernetes/kubernetes/pull/124598), [@aroradaman](https://github.com/aroradaman))",
```
```suggestion
    "markdown": "Kubectl describe service and ingress will now use endpointslices instead of endpoints ([#124598](https://github.com/kubernetes/kubernetes/pull/124598), [@aroradaman](https://github.com/aroradaman))",
```
```suggestion
    "markdown": "Kubectl describe service and ingress will now use endpointslices instead of endpoints ([#124598](https://github.com/kubernetes/kubernetes/pull/124598), [@aroradaman](https://github.com/aroradaman))",
```
```suggestion
    "markdown": "Kubectl describe service and ingress will now use endpointslices instead of endpoints ([#124598](https://github.com/kubernetes/kubernetes/pull/124598), [@aroradaman](https://github.com/aroradaman))",
```
```suggestion
    "markdown": "Kubectl describe service and ingress will now use endpointslices instead of endpoints ([#124598](https://github.com/kubernetes/kubernetes/pull/124598), [@aroradaman](https://github.com/aroradaman)) [SIG CLI and Network",
```

Reply:
This one is missing the `]`